fe2-android/.gitignore

```diff
@@ -0,0 +1,19 @@
+# IntelliJ
```

Comment:
you might want to merge master into your branch first, some of these things are already ignored in the root .gitignore